contracts/staking/StakingInfo.sol

```diff
@@ -404,8 +404,10 @@ contract StakingInfo {
             _contract,
             status
         ) = stakeManager.validators(validatorId);
-        reward += IStakeManager(_contract).validatorRewards();
-        _status = uint256(status);
+        if (_contract != address(0x0)) {
+            reward += IStakeManagerLocal(_contract).validatorRewards();
+            _status = uint256(status);
```

Comment:
Status should be outside of this if condition.

Reply:
done